README.md

```diff
@@ -2,7 +2,7 @@
 ![GitHub Discussions](https://img.shields.io/github/discussions/microsoft/vscode-mssql)
 [![codecov](https://codecov.io/github/microsoft/vscode-mssql/graph/badge.svg?token=NXLNAwJgRB)](https://codecov.io/github/microsoft/vscode-mssql)
 
-# MSSQL extension for Visual Studio Code
+# MSSQL extension for Visual Studio Code (Test)
```

Comment:
<img width="449" height="102" alt="Image" src="https://github.com/user-attachments/assets/84dc2142-ebf3-499c-bf96-00a9bbcdef0c" />

You can't tell me what to do.